### ISSUE TITLE
BR: time conclusion in summary is confusing

### ISSUE THREAD

**Issue by cyliu0** [pingcap/br#960]
Please answer these questions before submitting your issue. Thanks!

1. What did you do?
If possible, provide a recipe for reproducing the error.
BR restore 28k tables

2. What did you expect to see?
The time conclusion is reasonable

3. What did you see instead?
It has took ~5h, but there is `total take(Database restore time): 9h6m59.805780899s, total take(real time): 5h19m11.554582738s`. It's confusing for cusotmers
```
[2021/03/28 13:21:35.678 +00:00] [INFO] [collector.go:60] ["Database restore Success summary: total restore files: 101076, total success: 101076, total failed: 0, total take(Database restore time): 9h6m59.805780899s, total take(real time): 5h19m11.554582738s, total kv: 7912835, total size(MB): 586.40, avg speed(MB/s): 0.02"] ["split region"=4h39m41.038986486s] ["restore checksum"=5h19m5.839916585s] ["restore ranges"=101075] [Size=639231847]
```

4. What version of BR and TiDB/TiKV/PD are you using?

release-5.0

5. Operation logs
   - Please upload `br.log` for BR if possible
   - Please upload `tidb-lightning.log` for TiDB-Lightning if possible
   - Please upload `tikv-importer.log` from TiKV-Importer if possible
   - Other interesting logs

6. Configuration of the cluster and the task
   - `tidb-lightning.toml` for TiDB-Lightning if possible
   - `tikv-importer.toml` for TiKV-Importer if possible
   - `topology.yml` if deployed by TiUP

7. Screenshot/exported-PDF of Grafana dashboard or metrics' graph in Prometheus if possible

**Comment by YuJuncen** [comment]
Would be fixed after #970.

**Comment by YuJuncen** [comment]
Close it due to #970.
